quantum-adapters/kiwoom-adapter/src/kiwoom_api/realtime/client.py

```
@@ -175,36 +216,44 @@ async def send_message(self, message: Dict[str, Any]) -> bool:
+                    for rt_type in types:
                         if rt_type not in self.subscriptions[symbol]:
                             self.subscriptions[symbol].append(rt_type)
 
-                        logger.info(f"실시간 구독: {symbol} - {rt_type}")
-                        await asyncio.sleep(0.1)  # 요청 간격
-
-            return True
+                logger.info(f"✅ 실시간 구독 성공: {symbols} - {types}")
+                return True
+            else:
+                logger.error(f"❌ 실시간 구독 실패: 메시지 전송 오류")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)